assets/styl/_elements/_modal.styl

```diff
@@ -0,0 +1,43 @@
+// Modal
+//
+// Colette uses the accessible modal a11y-dialog (https://www.npmjs.com/package/a11y-dialog)
+//
+// Markup: modal.twig
+//
+// Styleguide Elements.Modal
+.modal
+    &[aria-hidden=true]
+        display none
+
+    &-overlay
+        position fixed
+        z-index: $zIndex.modal
+        top 0
+        left 0
+        width 100%
+        height 100%
+        background rgba(0, 0, 0, .6)
+
+    &-content
+        position fixed
+        z-index: $zIndex.modal
+        left 50%
+        top 50%
+        transform translate(-50%, -50%)
+        width fit-content
```

Comment:
https://caniuse.com/#search=fit-content
support of fit-content is not good enough